tests/h/views/client_test.py

```diff
@@ -24,26 +24,24 @@ def test_it_includes_client_config(self, pyramid_request):
 
         assert actual_config == expected_config
 
-    def test_it_sets_embed_url(self, pyramid_request):
+    def test_it_sets_client_url(self, pyramid_request):
         ctx = client.sidebar_app(pyramid_request)
 
-        assert ctx["embed_url"] == "/embed.js"
+        assert ctx["client_url"] == "http://example.com/client_url"
 
     def test_it_sets_custom_content_security_policy_header(self, pyramid_request):
         client.sidebar_app(pyramid_request)
         csp_header = pyramid_request.response.headers["Content-Security-Policy"]
 
         assert (
             csp_header
-            == "script-src 'self' http://example.com https://www.google-analytics.com; style-src http://example.com 'unsafe-inline'"
+            == "script-src http://example.com https://www.google-analytics.com; style-src http://example.com 'unsafe-inline'"
```

Comment:
I trust you here. I am not familiar with the Content-Security-Policy rules.

Reply:
Ah - there is something confusing about this test. http://example.com is a stand-in for the client's CDN (cdn.hypothes.is). The client no longer loads any scripts from https://hypothes.is so we can remove the "self" rule. We could also remove the Google Analytics rule here since the client doesn't use GA any more.